exercise_1/Graph.hpp

```diff
@@ -0,0 +1,37 @@
+//
+// Created by denys on 08.07.19.
+//
+
+#pragma once
+
+#include <list>
+#include <fstream>
+#include <regex>
+#include <map>
+#include "Node.hpp"
+
+
+class Graph {
```

Comment:
For additional exercise:
Could you rewrite this class to template which is specialized by Node type?

Reply:
I'll do that in second commit. Is that OK?

Reply:
OK

Reply:
Add templates (6a5ad78).